docs/cri/architecture.md

```diff
@@ -1,7 +1,7 @@
 # Architecture of The CRI Plugin
 This document describes the architecture of the `cri` plugin for `containerd`.
 
-This plugin is an implementation of Kubernetes [container runtime interface (CRI)](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto). Containerd operates on the same node as the [Kubelet](https://kubernetes.io/docs/reference/generated/kubelet/). The `cri` plugin inside containerd handles all CRI service requests from the Kubelet and uses containerd internals to manage containers and container images.
+This plugin is an implementation of Kubernetes [container runtime interface (CRI)](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto). Containerd operates on the same node as the [Kubelet](https://kubernetes.io/docs/reference/generated/kubelet/). The `cri` plugin inside containerd handles all CRI service requests from the Kubelet and uses containerd internals to manage containers and container images.
 
 The `cri` plugin uses containerd to manage the full container lifecycle and all container images. As also shown below, `cri` manages pod networking via [CNI](https://github.com/containernetworking/cni) (another CNCF project).
 
```

Comment:
Please update this too: https://github.com/containerd/containerd/blob/main/RELEASES.md#kubernetes-support

Reply:
Should I update the "Kubernetes Support" section like the following?

```
| Kubernetes Version | containerd Version | CRI Version  |
|--------------------|--------------------|--------------|
| 1.20               | 1.5.0+             | v1alpha2     |
| 1.21               | 1.5.0+             | v1alpha2     |
| 1.22               | 1.5.5+             | v1alpha2     |
| 1.23               | 1.6.0+, 1,5.8+     | v1, v1alpha2 |
| 1.24               | 1.6.4+, 1.5.11+    | v1, v1alpha2 |
| 1.25               | 1.6.4+, 1.5.11+    | v1, v1alpha2 |
| 1.26               | ???                | v1 |
```

I assume this change will go to 1.7 so the containerd version here should be 1.7.0+?

Reply:
Thanks, looks good

Reply:
right but with 1.6 being the LTS version, I guess we want to include 1.6.x for 1.26 as well? If thats the case do we know which patch version of 1.6.x should we use here?

Reply:
Yes, and I guess `1.6.4+` is fine, assuming that the CRI v1 implementation of K v1.26 is compatible with K v1.25 and v1.24?